### DISCUSSION TITLE
脑控在中国

### DISCUSSION THREAD

**Discussion by gangstalking1984** [WeNeedHome/SummaryOfLoanSuspension#990] [General]
![Screenshot_20220723_045203](https://user-images.githubusercontent.com/109858971/180606591-c49aade5-98e9-40ce-a400-af96d15fe60a.jpg)
![XHS_165573673139901655736907106](https://user-images.githubusercontent.com/109858971/180606611-c8346e5c-ab4a-42ea-ad03-d415b5e8e044.jpg)
![XHS_165573673139911655736907271](https://user-images.githubusercontent.com/109858971/180606633-d9c3ec7b-d6f9-4c97-90b3-752dd69c114c.jpg)
![XHS_165573673139921655736907419](https://user-images.githubusercontent.com/109858971/180606637-9a0899eb-e8cc-4634-a347-c9451255e26e.jpg)

**Comment by gangstalking1984** [comment]
![XHS_165573673139931655736907502](https://user-images.githubusercontent.com/109858971/180606714-1e512d57-5a1c-4fb4-ac51-9c501f00862e.jpg)

**Reply by gangstalking1984** [reply]
![XHS_164417931675141644179493831](https://user-images.githubusercontent.com/109858971/180606814-0dade8ca-3ef6-4444-a085-4735efcd303d.jpg)
![XHS_164417931675151644179493882](https://user-images.githubusercontent.com/109858971/180606820-b9227fa5-2aae-4198-a876-3253eac39ac2.jpg)
![XHS_164365370379001643653792567](https://user-images.githubusercontent.com/109858971/180606825-537c077e-e3b8-45a7-a4af-86ea9e934dbf.jpg)

**Comment by gangstalking1984** [comment]
![1655606039828](https://user-images.githubusercontent.com/109858971/180607105-2cbd3fe7-7690-4493-8d7a-e5fbf76d4876.jpg)
![XHS_164412512908701644125307326](https://user-images.githubusercontent.com/109858971/180607110-7ca6a19c-cd69-4995-b329-6acd4a6c5d57.jpg)
![XHS_164412512908711644125307481](https://user-images.githubusercontent.com/109858971/180607113-1908b3f8-c732-4ae2-afa1-83ef6921bf63.jpg)
![XHS_164092632994631640926414261](https://user-images.githubusercontent.com/109858971/180607115-548fb821-f2cc-4572-9484-57e2f960242a.jpg)
![XHS_164041470720361640414822390](https://user-images.githubusercontent.com/109858971/180607117-113d99ee-4184-4f31-812a-a10d0b0c2c13.jpg)

**Reply by gangstalking1984** [reply]
![Screenshot_20220727_193103](https://user-images.githubusercontent.com/109858971/181295299-a546468a-434c-49ab-a066-ed43ae04dabf.jpg)
![wx3 sinaimg](https://user-images.githubusercontent.com/109858971/181295311-9662877f-67c7-4394-974b-1bf8f14bdfc6.jpeg)

**Comment by Shenkeis** [comment]
这TM是什么品种的弱智。。
老实说 你是不是在抄袭 红警2尤里的复仇剧情？

**Reply by gangstalking1984** [reply]
联合国承认脑控犯罪的存在了，命名为网络酷刑

**Reply by gangstalking1984** [reply]
> 这TM是什么品种的弱智。。 老实说 你是不是在抄袭 红警2尤里的复仇剧情？

大家注意，这是条五毛，混淆视听掩盖真相的，他们最拿手的就是扣帽子和人身攻击

**Reply by gangstalking1984** [reply]
你们共产党已经没有底线了！什么事情都干的出来！养你们这些水军爪牙就可以掩盖真相吗？看看这几年你们干了什么！拿无辜平民做人体实验！在中国投放病毒让几百万老人死亡！杀人拐卖儿童偷取器官！不要以为有媒体和水军帮你们掩盖就可以只手遮天！

**Comment by Cegazhos** [comment]
标记为顶真

**Comment by Kamijoucen** [comment]
我一个神神都觉得你太弱智了😅

**Reply by gangstalking1984** [reply]
你们这些五毛爪牙休得猖狂，以为你们人多就可以瞒天过海，人在做天在看，亏心事干多了要遭报应的

**Comment by Ventery** [comment]
把大伙都整笑了

**Comment by gangstalking1984** [comment]
你们共产党已经没有底线了！什么事情都干的出来！养你们这些水军爪牙就可以掩盖真相吗？看看这几年你们干了什么！拿无辜平民做人体实验！在中国投放病毒让几百万老人死亡！杀人拐卖儿童偷取器官！不要以为有媒体和水军帮你们掩盖就可以只手遮天！
